IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDWebviewUIController.m

```diff
@@ -91,6 +91,8 @@ - (BOOL)loadView:(__unused NSError **)error
 {
     if (_webView)
     {
+        _loadingIndicator = [self prepareLoadingIndicator];
+        [_webView addSubview:_loadingIndicator];
```

Comment:
Should the loading indicator be added to root view instead of _webView like in line 115?